scripts/commands/staffCommands.js

```diff
@@ -256,6 +256,59 @@ const registerCommands = (clientCommands, serverCommands, runner) => {
     })
   );
 
+  clientCommands.register(
+    'stop_offline',
+    '<name>',
+    'Stops an offline player.',
+    runner((args, realP) => {
+      const p = players.getP(realP);
+      const admins = Vars.netServer.admins;
+
+      if (p.fakeAdmin) {
+        return;
+      }
+
+      if (!p.mod && !p.admin) {
+        realP.sendMessage('[scarlet]⚠ [yellow]You do not have access to this command.');
+        return;
+      }
+      let possiblePlayers = admins.searchNames(args[0]);
+      if(possiblePlayers.length > 20){
```

Comment:
This is wrong, ObjectSet does not have a length attribute

Reply:
Is this good to merge or no?

Reply:
It "works" but I should fix it